### ISSUE TITLE
Frozen maps, support status

### ISSUE THREAD

**Issue by hemidactylus** [stargate/data-api#1668]
Right now frozen maps cannot be read off a table (e.g. one created in direct CQL)

```
"Server failed: root cause: (java.lang.IllegalStateException) Unsupported columns in the result set: m(UNSUPPORTED)"
```

Likewise,

```
                        "m": {
                            "type": "UNSUPPORTED",
                            "apiSupport": {
                                "createTable": false,
                                "insert": false,
                                "read": false,
                                "cqlDefinition": "frozen<map<text, text>>"
                            }
                        }
```

but this looks like eventually it should be readable. Are there plans?

**Comment by Yuqi-Du** [comment]
reproducing
Table:
```
CREATE TABLE filter.orders (
    order_id text PRIMARY KEY,
    customer_name text,
    item_details frozen<map<text, int>>,
    items frozen<list<text>>
)
```

```
{
    "errors": [
        {
            "message": "Server failed: root cause: (java.lang.IllegalStateException) Unsupported columns in the result set: item_details(UNSUPPORTED), items(UNSUPPORTED)",
            "errorCode": "SERVER_UNHANDLED_ERROR",
            "id": "d15945ee-d633-4782-9711-d4a5d246e728",
            "family": "SERVER",
            "title": "Server failed",
            "scope": "DATABASE"
        }
    ]
}
```

**Comment by Yuqi-Du** [comment]
should not be 500, need better error handling.

**Comment by amorton** [comment]
fix: should be able to read a frozen map but not update, related to #1568

**Comment by amorton** [comment]
will fix in #1568
